src/renderer/lib/torrent-poster.js

```diff
@@ -110,6 +110,8 @@ function scoreAudioCoverFile (imgFile) {
 function torrentPosterFromAudio (torrent, cb) {
   const imageFiles = filterOnExtension(torrent, mediaExtensions.image)
 
+  if (imageFiles.length === 0) return cb(new Error('Generated poster contains no files'))
```

Comment:
I suppose it can. Since we caught the degenerate case of an empty array and if the array only has 1 item, the callback doesn't even get call. So there should be no reason to for `bestCover` to be falsey. If there's a issue with each file item, then the issue is probably further upstream.

Reply:
Okay, thanks for fixing it 👍